packages/grafana-toolkit/src/cli/tasks/plugin.dev.ts

```diff
@@ -2,11 +2,36 @@ import { Task, TaskRunner } from './task';
 import { bundlePlugin as bundleFn, PluginBundleOptions } from './plugin/bundle';
 import { useSpinner } from '../utils/useSpinner';
 
+// @ts-ignore
+import execa = require('execa');
+import path = require('path');
+
 const bundlePlugin = useSpinner<PluginBundleOptions>('Bundling plugin in dev mode', options => {
   return bundleFn(options);
 });
 
+const yarnlink = useSpinner<void>('Linking local toolkit', async () => {
+  // await execa('yarn', ['remove', '@grafana/toolkit']);
```

Comment:
@dprokop -- this fails if `@grafana/toolkit` is not in your package.json.  Ideas on how to best avoid that?  adding `--force` does not help.  maybe need to catch an exception from execa and ignore it?

Reply:
Fixed in 3ed6163a0a30ac20c023f15ed91c07234d714705 (try/catch was sufficient)